services/download/src/main/java/app/coronawarn/server/services/download/BatchDownloadException.java

```diff
@@ -1,12 +1,19 @@
 package app.coronawarn.server.services.download;
 
+import static java.time.format.DateTimeFormatter.ISO_LOCAL_DATE;
+
+import java.time.LocalDate;
+
 public class BatchDownloadException extends RuntimeException {
```

Comment:
Now seeing this: Maybe we should change the type to a checked exception? Because we want to make sure that we catch every instance.